### PR TITLE
#353 - Auth tests joined for endpoints requiring write permissions

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #353 
Auth tests joined for endpoints requiring write permissions

### CODE REVIEW

File: src/test/java/com/artipie/docker/http/AuthTest.java [hunk 1]
```diff
@@ -121,17 +131,69 @@ void shouldNotReturnUnauthorizedOrForbiddenWhenUserHasPermissions(final RequestL
...
-            new RequestLine(RqMethod.HEAD, "/v2/my-alpine/manifests/1"),
-            new RequestLine(RqMethod.GET, "/v2/my-alpine/manifests/2"),
-            new RequestLine(RqMethod.GET, "/v2/my-alpine/tags/list"),
-            new RequestLine(RqMethod.GET, "/v2/my-alpine/blobs/uploads/112233"),
-            new RequestLine(RqMethod.GET, "/v2/_catalog")
-        ).map(Arguments::of);
+            Arguments.of(
+                new RequestLine(RqMethod.GET, "/v2/"),
+                TestAuthentication.ALICE
+            ),
```
Comment (genryxy) [inline src/test/java/com/artipie/docker/http/AuthTest.java:152]: @olegmoz I'd suggest to extract two methods, one for `Alice` and another for `Bob`. And pass RqMethod and path to these methods.

**Review by genryxy — CHANGES_REQUESTED** [review]
@olegmoz thanks, one suggestion

**Comment by olegmoz** [comment]
@genryxy thanks, nice suggestion. Split the endpoints by read/write

**Review by genryxy — APPROVED** [review]
@olegmoz thanks, approve

**Pull request merged**